etl/extractor/platform_impl.go

```diff
@@ -224,11 +236,18 @@ func (e *PlatformExtractor) retrieveRecords(ctx context.Context, pu *exporter.Fu
 			if err == io.EOF { // stream ended, we have our portion
 				break
 			}
+			if err != nil && isRateLimitError(err) {
+				log.Error("retrieveRecords() on rpc call: ", err.Error())
+				Errors.With(ErrorTypeRateLimitExceeded).Inc()
+				closeStream(cancelCtx, stream)
+				time.Sleep(halfPulse)
+				break
```

Comment:
would you mind to left here a comment why we need to reopen the stream?

Reply:
done